Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -1580,14 +1580,17 @@ class CloneSerializer : public CloneBase {
 
     void dumpImageBitmap(JSObject* obj, SerializationReturnCode& code)
     {
+        auto& imageBitmap = jsCast<JSImageBitmap*>(obj)->wrapped();
         auto index = m_transferredImageBitmaps.find(obj);
         if (index != m_transferredImageBitmaps.end()) {
+#if USE(SKIA)
```

Comment:
A bit unfortunate to add `#if USE(SKIA)` guards here. But I don't know a better solution right now. Eventually this can be made cross-plaform and a no-op for non-Skia platforms?

Reply:
if more platforms use those functions one day then I think we can do it so